.github/workflows/ci-cd.yml

```diff
@@ -32,8 +32,7 @@ jobs:
         id: vars
         shell: bash
         run: |
-          msys2_extras=$(grep -o 'pacman --noconfirm -Syu --needed [^"]*' tasks.star | cut -d ' ' -f 5-)
-          msys2_extras=$(sh -c "echo $msys2_extras") # subshell echo to perform brace expansion
+          msys2_extras=$(cat msys2-packages.txt)
```

Comment:
You can actually write this `$(< file to read)`, which saves on a useless use of cat! Pretty nifty.

Reply:
Nice! If the CI passes, I'll add that change. If it doesn't, I'll add it to whatever I'll have to fix.